app/views/providers/update_email_addresses/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= page_template(
+      page_title: t('.page_title'),
+      back_link: { text: t('generic.back') },
+      page_heading_options: { margin_bottom: 3 }
+    ) do %>
+
+  <%= form_with(
+        model: @form,
+        url: providers_legal_aid_application_update_email_address_path,
+        method: :patch,
+        local: true
+      ) do |form| %>
+
+    <div class="govuk-!-padding-bottom-6"></div>
+
+    <%= form.govuk_text_field(
+          :email,
+          label: { text: t('providers.applicants.show.email_label'), size: :m },
+          hint: t('helpers.hint.applicant.email'), size: :m,
+          class: 'govuk-input govuk-!-width-three-quarters'
+        ) %>
```

Comment:
`govuk_text_field` by default grabs the label from `activemodel.attributes.applicant.email` and the hint from `helpers.hint.applicant.email`. And they happen to be exactly what you need.

Also, by default the class `govuk-input` is always applied.

So you can just write:
```ruby
    <%= form.govuk_text_field(
          :email,
          label: { size: :m },
          class: 'govuk-!-width-three-quarters'
        ) %>
```